packages/@webex/plugin-meetings/src/reachability/index.ts

```diff
@@ -430,6 +456,29 @@ export default class Reachability {
     });
   }
 
+  /**
+   * Set client media IPs
+   * @param {RTCPeerConnection} peerConnection
+   * @param {string} clientMediaIP
+   * @returns {void}
+   */
+  private setClientMediaIPs(peerConnection: RTCPeerConnection, clientMediaIP?: string) {
+    const {CLOSED} = CONNECTION_STATE;
+
+    if (peerConnection.connectionState === CLOSED) {
+      LoggerProxy.logger.log(
+        `Reachability:index#setClientMediaIPs --> Attempting to set clientMediaIP of ${clientMediaIP} on closed peerConnection.`
+      );
+    }
+
+    // Set to null in case this fired from
+    // an event other than onIceCandidate
+    peerConnection.onicecandidate = null;
+    peerConnection.close();
+    // @ts-ignore
+    peerConnection.clientMediaIP = clientMediaIP;
```

Comment:
there can be multiple of these IPs  (if the computer has multiple network adapters connected to different networks), Orpheus wants to know all of them. So it probably also makes sense to rename setClientMediaIPs() to addClientMediaIPs() or actually addPublicIP() (see my other comment)

Reply:
ooh i seeee. i've kind of tried to add the "multiple IPs" logic in the `parseIceResultsToReachabilityResults`, but it was wrong. Makes more sense here now, I was really confused yesterday..